news/3821.feature.rst

```diff
@@ -0,0 +1 @@
+Allow the user to prevent binaries from being compressed with UPX, using `--exclude-upx`
```

Comment:
I'd start with "New command-line option ``--exclude-upx``" (mind therst-Syntax: douple-`.
Trailing period is missing.